core/unit_tests.c

```diff
@@ -59,6 +60,10 @@ void
 unit_test_atomic_ops(void);
 void
 unit_test_jit_fragment_tree(void);
+void
+unit_test_get_ymm_caller_saved(void);
+void
+unit_test_get_zmm_caller_saved(void);
```

Comment:
No reason to reference these or call these here: these are private to x86_code.c and should be called by unit_test_asm() in that file.

Reply:
Done.